canonical/ubuntu.com
# add pro/beta page

## Done

- added /pro/beta, according to the [copy doc](https://docs.google.com/document/d/1GFVENQ9g5U4FSyqZHboXVEQ_RHKxRSwq3NC9PWrwURs/edit#) and the [design](https://www.figma.com/file/ZAihKxF2wmj9wvoPSnPKmY/Commercial%3A-UA-%2Fsubscribe?node-id=1571%3A14153)

## QA

- Visit https://ubuntu-com-12099.demos.haus/pro/beta
- See that it matches the design and the copy doc

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/6046
